src/libs/de-sdk/utils/intents.ts

```diff
@@ -1,5 +1,6 @@
 import { parseUnits } from "viem"
 import * as borsh from "borsh"
+import { Ojuju } from "next/dist/compiled/@next/font/dist/google"
```

Comment:
What is it ser ? @jobotics

Reply:
Probably auto import from my IDEI client, I will remove it. Thanks!

Reply:
Probably auto import from my IDEI client, I will remove it. Thanks!